src/main/models/job.ts

```diff
@@ -64,6 +64,15 @@ class JobDb extends Model<
     });
   }
 
+  public static async testGetJobByUid(uid: string) {
```

Comment:
What is this for?

Reply:
My bad, left something in that I was using for testing the change. Removed.